ui/00_message/tutorial_guide/tutorial_guide.xml

```diff
@@ -627,6 +635,7 @@
   <message>
     <key>TUTORIAL_255</key>
     <original>鑑定交換</original>
+	<translation>Appraisal Exchange</translation>
```

Comment:
Yeah thats weird, I specifically use four spaces in these cases. Ill need to see whats up